<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysDetailsWidget</class>
 <widget class="QWidget" name="SysDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="IndicatorWidget" name="sysDetailsIndicator" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="sysUpdatesStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QTreeWidget" name="sysUpdatesList">
         <column>
          <property name="text">
           <string>Update</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Update type</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="sysInstallSelectedBtn">
           <property name="text">
            <string>Install selected only</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/install.png</normaloff>:/images/install.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="sysUpdateDetailsStack">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_9">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="res.qrc">:/images/sysupdates-patch.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>PC-BSD update patch</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Description:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="sysPadthDescription">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="textInteractionFlags">
                    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="openDeskriptionBtn">
                   <property name="toolTip">
                    <string>Open description on web browser</string>
                   </property>
                   <property name="text">
                    <string>...</string>
                   </property>
                   <property name="icon">
                    <iconset resource="res.qrc">
                     <normaloff>:/images/internet-web-browser.png</normaloff>:/images/internet-web-browser.png</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Date:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="sysPatchDate">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Size:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="sysPatchSize">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="sysPatchStandalone">
               <property name="text">
                <string>Update should be installed standalone. </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sysPatchRebootRequired">
               <property name="text">
                <string>Update requires system reboot</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>314</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_10">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="res.qrc">:/images/sysupdates-upgrade.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>PC-BSD upgrade</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sysUpgradeText">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sysUpgradeStandalone">
               <property name="text">
                <string>Update should be installed standalone. </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sysUpgradehRebootRequired">
               <property name="text">
                <string>Update requires system reboot</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>347</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_11">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="res.qrc">:/images/sysupdates-freebsd.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTabWidget" name="sysUpdateFilesTab">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_4">
              <attribute name="title">
               <string>Files to update</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QListWidget" name="sysFilesToUpdateList"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_5">
              <attribute name="title">
               <string>Files to delete</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QListWidget" name="sysFilesToDeleteList"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_6">
              <attribute name="title">
               <string>Locally modified files</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QListWidget" name="sysFilesModifyedList"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>System update log:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="sysUpdateLog">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IndicatorWidget</class>
   <extends>QWidget</extends>
   <header location="global">indicatorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
